yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -50,7 +50,7 @@ func (d *TableIssueDetector) DetectIssues(obj queryparser.DDLObject) ([]QueryIss
 	// Check for generated columns
 	if len(table.GeneratedColumns) > 0 {
 		issues = append(issues, NewGeneratedColumnsIssue(
-			TABLE_OBJECT_TYPE,
+			obj.GetObjectType(),
```

Comment:
a `TableIssueDetector` will always return issues for Table object types, right? 
Is this just a change to not use a constant and use the function, or is there a certain bug/case because of which you had to do this?

Reply:
> Is this just a change to not use a constant and use the function,

Yes right, that way I was able to directly use `ddlObj.GetObjectType()` to get the respective type.